nerfstudio/scripts/datasets/process_project_aria.py

```diff
@@ -184,59 +292,126 @@ class ProcessProjectAria:
     """Path to Project Aria Machine Perception Services (MPS) attachments."""
     output_dir: Path
     """Path to the output directory."""
+    points_file: Optional[Path] = None
+    """Path to the point cloud file (usually called semidense_points.csv.gz) if not in the mps_data_dir"""
```

Comment:
just wondering, when is the point cloud file to not in the MPS dir?

Reply:
When I downloaded the Aria Everyday Activities Dataset, the pointcloud was in its own separate folder so i went ahead and added support for both cases

Reply:
aha that makes sense!

Reply:
is it one shared point cloud across all users?

Reply:
actually it's multiple point cloud files, so rn main() will combine all of them since they should be using the same point of reference